.eslintrc

```diff
@@ -38,8 +38,6 @@
     "import/no-relative-packages": "error",
     "import/export": "error",
     "import/no-named-as-default": "error",
-    // TODO enable and fix
-    // "import/no-deprecated": "error",
```

Comment:
`import/no-deprecated` causes lot of false-positive. Actually, it gave nothing pertinent compared to `deprecation/deprecation`.